src/declarations/backend/backend.did.d.ts

```diff
@@ -15,6 +15,7 @@ export interface CanisterStatusResultV2 {
 	module_hash: [] | [Uint8Array | number[]];
 }
 export type CanisterStatusType = { stopped: null } | { stopping: null } | { running: null };
+export type CustomToken = { Icrc: IcrcToken };
```

Comment:
Just for my understanding:  `CustomToken` is defined as an enum in Rust.  Would a Rust enum be expressed as a union in typescript?  https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types  Thus if in Rust we have:

```
/// Rust enum
pub enum CustomToken {
    Icrc(IcRcToken),
    SomeOtherStandard(SomeOtherStandard),
} 
```
then in typescript this would be:
```
export type CustomToken = { Icrc: IcrcToken } | { SomeOtherStandard: whatever };
```
?

Reply:
That's correct, Candid parse Rust enum to union types in JS/TS.